### ISSUE TITLE
Terraform init failing for terraform version < 1.X

### ISSUE THREAD

**Issue by junaid-ali** [terraform-aws-modules/terraform-aws-eks#2126]
## Description

This module no longer seems to be working for terraform < `1.0`
```shell
$ terraform init -upgrade
Upgrading modules...
Downloading terraform-aws-modules/eks/aws 18.26.0 for eks...
- eks in .terraform/modules/eks
- eks.eks_managed_node_group in .terraform/modules/eks/modules/eks-managed-node-group
- eks.eks_managed_node_group.user_data in .terraform/modules/eks/modules/_user_data
- eks.fargate_profile in .terraform/modules/eks/modules/fargate-profile
Downloading terraform-aws-modules/kms/aws 1.0.0 for eks.kms...
- eks.kms in .terraform/modules/eks.kms
- eks.self_managed_node_group in .terraform/modules/eks/modules/self-managed-node-group
- eks.self_managed_node_group.user_data in .terraform/modules/eks/modules/_user_data
....

Error: Unsupported Terraform Core version

  on .terraform/modules/eks.kms/versions.tf line 2, in terraform:
   2:   required_version = ">= 1.0"

Module module.eks.module.kms (from terraform-aws-modules/kms/aws) does not
support Terraform version 0.14.11. To proceed, either choose another supported
Terraform version or update this version constraint. Version constraints are
normally set for good reason, so updating the constraint may lead to other
errors or unexpected behavior.
```

It could be due to the `kms` module which was added in https://github.com/terraform-aws-modules/terraform-aws-eks/compare/v18.24.1...v18.25.0

- [x] ✋ I have searched the open/closed issues and my issue is not listed.

## ⚠️ Note

Before you submit an issue, please perform the following first:

1. Remove the local `.terraform` directory (! ONLY if state is stored remotely, which hopefully you are following that best practice!): `rm -rf .terraform/`
2. Re-initialize the project root to pull down modules: `terraform init`
3. Re-attempt your terraform plan or apply and check if the issue still persists

## Versions

- Module version [Required]: `18.26.0`

- Terraform version: `0.14.6`

- Provider version(s): `aws v4.20.1`

## Reproduction Code [Required]

1. Remove the `.terraform` directory
2. Run `terraform init` with a terraform version less `1.x`

**Comment by bryantbiggs** [comment]
oof, thanks for reporting. I'll have a fix shortly

**Comment by antonbabenko** [comment]
This issue has been resolved in version 18.26.1 :tada:
